src/python/pants/backend/terraform/lint/tfsec/tfsec.py

```diff
@@ -24,7 +24,10 @@ class TFSec(ExternalTool):
     help = "tfsec by Aqua Security"
     default_version = "v1.28.1"
     default_known_versions = [
-        "v1.28.1|linux_x86_64|57b902b31da3eed12448a4e82a8aca30477e4bcd1bf99e3f65310eae0889f88d|26427634"
+        "v1.28.1|linux_x86_64|57b902b31da3eed12448a4e82a8aca30477e4bcd1bf99e3f65310eae0889f88d|26427634",
```

Comment:
This was simply missing for all these platforms, so did not work on them until now.

Reply:
Hmm, I guess technically this is a bugfix. Maybe I will put this in a separate PR.